doc/06_focusing_and_skipping.md

```diff
@@ -30,9 +35,9 @@ or namespace metadata, based on four command line flags and configuration keys.
 
 ## Matching
 
-### On id
+### by var
```

Comment:
I would leave this title, since not all test types have vars (e.g. cucumber), but all tests have ids.